### PR TITLE
PP-8947 - Add FeeType field to FeeEntity

### PR DESCRIPTION
**Pull request by nimalank7**
Description:
- Adds FeeType field to FeeEntity object for the [database migration](https://github.com/alphagov/pay-connector/pull/3323)
- Removed some unused imports whilst working on the PR
- Paired with @SandorArpa

### CODE REVIEW

File: src/main/java/uk/gov/pay/connector/charge/model/domain/FeeType.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package uk.gov.pay.connector.charge.model.domain;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+
+public enum FeeType {
+    RADAR("radar"),
+    THREE_D_S("3ds"),
```
Comment (kbottla) [inline src/main/java/uk/gov/pay/connector/charge/model/domain/FeeType.java]: wondering if the value should be `three_d_secure` or `three_ds` to work nicely with javascript. Although json allows it, we could get caught if this is used by javascript app (selfservice). We can't use fee.3ds and instead need to use fee['3ds'] where as other can be accessed by dot notation

File: src/main/java/uk/gov/pay/connector/charge/model/domain/FeeTypeConverter.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package uk.gov.pay.connector.charge.model.domain;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+@Converter
+public class FeeTypeConverter implements AttributeConverter<FeeType, String> {
```
Comment (kbottla) [inline src/main/java/uk/gov/pay/connector/charge/model/domain/FeeTypeConverter.java:7]: Can we add tests for this ?

File: src/main/java/uk/gov/pay/connector/charge/model/domain/FeeType.java (hunk 1 above)
Comment (nimalank7) [reply src/main/java/uk/gov/pay/connector/charge/model/domain/FeeType.java]: Done.

File: src/main/java/uk/gov/pay/connector/charge/model/domain/FeeTypeConverter.java (hunk 2 above)
Comment (nimalank7) [reply src/main/java/uk/gov/pay/connector/charge/model/domain/FeeTypeConverter.java:7]: Added.

**Review by kbottla — APPROVED** [review]
LGTM!

**Pull request merged**